app/Jobs/RestoreDatabaseJob.php

```
@@ -0,0 +1,193 @@
+                'database' => $this->database->uuid,
+                'backup_label' => $this->backupLabel,
+                'target_time' => $this->targetTime,
+                'delta' => $this->delta,
+                'job_id' => $this->jobId,
+            ]);
+
+            $server = $this->database->destination->server;
+            $team = $this->database->team();
+
+            if (! $server) {
+                throw new \Exception('Server not found for database');
```

PHPMD (2.15.0)

Missing class import via use statement (line '53', column '27'). (undefined)

(MissingImport)